ml-agents/mlagents/trainers/tests/test_config_conversion.py

```diff
@@ -27,6 +27,8 @@
         lambd: 0.95
         learning_rate: 3.0e-4
         learning_rate_schedule: linear
+        beta_schedule: constant
+        epsilon_schedule: linear
```

Comment:
Is it possible to add some assertion at here to check the default replacement of beta_schedule and epsilon_schedule?